### ISSUE TITLE
package.json missing jquery

### ISSUE THREAD

**Issue by Spechal** [codeforkansascity/meep#60]
npm install reports jquery > 1.9.1 is required by bootstrap but it not included.  Including jquery 3.4.1 in pacakge.json resolves the issue.

**Comment by Spechal** [comment]
```
$> npm install
npm WARN bootstrap@4.3.1 requires a peer of jquery@1.9.1 - 3 but none is installed. You must install peer dependencies yourself.
npm WARN css-loader@2.1.0 requires a peer of webpack@^4.0.0 but none is installed. You must install peer dependencies yourself.
```

**Comment by Spechal** [comment]
Resolved in #67
